src/pages/correspondent/Forms/CorrespondentForm.js

```diff
@@ -266,6 +271,38 @@ const CorrespondentForm = ({ labels, editMode, maxAccess, setEditMode, setStore,
                 error={formik.touched.iwRateTypeId && Boolean(formik.errors.iwRateTypeId)}
               />
             </Grid>
+            <Grid item xs={12}>
+              <ResourceLookup
+                endpointId={SystemRepository.NumberRange.snapshot}
+                valueField='reference'
+                displayField='description'
+                name='nraId'
+                valueShow='nraRef'
+                secondValueShow='nraDescription'
+                label={labels.nuRange}
+                form={formik}
+                secondDisplayField={true}
+                firstValue={formik.values.nraRef}
+                secondValue={formik.values.nraDescription}
+                onChange={(event, newValue) => {
+                  formik.setFieldValue('nraId', newValue?.recordId || null)
+                  formik.setFieldValue('nraRef', newValue?.reference || null)
+                  formik.setFieldValue('nraDescription', newValue?.description || null)
+                }}
+                errorCheck={'nraId'}
+                maxAccess={maxAccess}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <CustomNumberField
+                name='minReviewTime'
+                label={labels.minReviewTime}
+                value={formik.values.minReviewTime}
+                onChange={formik.handleChange}
+                onClear={() => formik.setFieldValue('minReviewTime', '')}
+                error={formik.touched.minReviewTime && Boolean(formik.errors.minReviewTime)}
+              />
+            </Grid>
             <Grid item xs={12}>
               <FormControlLabel
                 control={
```

Comment:
![image](https://github.com/user-attachments/assets/7e8867ea-2a5b-47e8-8369-e58e9c4790f1)
i tried to clear the account ref , and i got duplicate of reference